src/alfasim_sdk/_internal/version.py

```diff
@@ -1 +1 @@
-__version__ = "2024.1"
+__version__ = "2024.2"
```

Comment:
How about using a `.dev` suffix to signal this has not been released yet?

```suggestion
__version__ = "2024.2.dev"
```

Reply:
OK... I will update the alfasim release process to mention we should update the sdk version when starting an alfasim RB.